download_data.py

```diff
@@ -220,18 +250,19 @@ def download_data(dataset,
     parser.add_argument(
         "--raw_dataset_assets",
         nargs='+',
-        default=['mov', 'annotation', 'mesh', 'confidence', 'highres_depth', 'lowres_depth',
-                 'lowres_wide.traj', 'lowres_wide', 'lowres_wide_intrinsics', 'ultrawide',
-                 'ultrawide_intrinsics', 'vga_wide', 'vga_wide_intrinsics']
+        default=default_raw_dataset_assets
     )
 
     args = parser.parse_args()
-    if args.video_id is None and args.video_id_csv is None:
-        raise argparse.ArgumentError('video_id or video_id_csv must be specified')
-    elif args.video_id is not None and args.video_id_csv is not None:
-        raise argparse.ArgumentError('only video_id or video_id_csv must be specified')
-    if args.video_id is not None and args.split is None:
-        raise argparse.ArgumentError('given video_id the split argument must be specified')
+    for asset in args.raw_dataset_assets:
+        assert asset in default_raw_dataset_assets, \
```

Comment:
Please remove this assets - Redundant once `choices` is used as part of the asset parsing.